backend/src/main/java/capstoneexpensetracker/backend/bill/BillController.java

```diff
@@ -1,21 +1,22 @@
 package capstoneexpensetracker.backend.bill;
 
 import lombok.RequiredArgsConstructor;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
 @RestController
 @RequestMapping("api/bill")
 @RequiredArgsConstructor
 public class BillController {
 
-
     private final BillService billService;
 
     @GetMapping("{travelerGroupId}")
     Bill getGroupBill(@PathVariable String travelerGroupId) {
         return billService.getBillByGroupId(travelerGroupId);
     }
+
+    @PutMapping("{groupId}/traveller/{travellerId}/amount")
+    Bill updateGroupBill(@PathVariable String groupId, @PathVariable String travellerId, @RequestBody UpdateAmount amountData) {
+        return billService.updateGroupBill(groupId, travellerId, amountData.amount());
```

Comment:
maybe catch the thrown errors from service, so that clients get correct Http-codes